sdk/tests/conformance2/extensions/ext-texture-norm16.html

```diff
@@ -73,14 +74,16 @@
   // Texture sampled from
   gl.activeTexture(gl.TEXTURE0);
   gl.bindTexture(gl.TEXTURE_2D, textures[0]);
-  gl.texImage2D(gl.TEXTURE_2D, 0, internalFormat, 1, 1, 0, format, type, imageData);
+  gl.texImage2D(gl.TEXTURE_2D, 0, ext[internalFormat] || gl[internalFormat],
+        1, 1, 0, gl[format], gl[type], imageData);
 
-  wtu.glErrorShouldBe(gl, gl.NO_ERROR, "texture bindings succeed");
+  wtu.glErrorShouldBe(gl, gl[error], `texImage should generate error:${error}`);
```

Comment:
To ensure that implementations enable the extension for all affected entry points, could other `texImage` functions be added here and throughout? Could we also test `renderbufferStorageMultisample` in `testNorm16Render`?

Reply:
I spent a few minutes on it but it was taking more time than I have, so follow-ups welcome.